cfgov/unprocessed/apps/paying-for-college/js/views/school-view.js

```diff
@@ -139,6 +163,36 @@ const schoolView = {
 
   },
 
+  _updateProgramList: () => {
+    let level = 'undergrad';
+    if ( getStateValue( 'programType' ) === 'graduate' ) {
+      level = 'graduate';
+    }
+    const list = getProgramList( level );
+    if ( list.length > 0 ) {
+      updateState.byProperty( 'schoolHasPrograms', 'yes' );
+    } else {
+      updateState.byProperty( 'schoolHasPrograms', 'no' );
+    }
+
+    if ( list.length > 0 ) {
+      let html = '<option selected="selected" value="null">Select...</option>';
+      list.forEach( elem => {
+        html += '\n<option data-program-salary="' + elem.salary + '"';
+        html += ' value="' + elem.code + '">';
+        html += elem.level + ' - ' + elem.name;
+        html += '</option>';
```

Comment:
Instead of the multiple lines of `html +=` you could use a [multi-line template literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#Multi-line_strings) to span multiple lines and write to the `html` variable once.

Reply:
Good call!